app/models/merchant.rb

```diff
@@ -36,4 +36,8 @@ def deactivate_items
     items.update(active?: false)
   end
 
+  def activate_items
+    items.update(active?: true)
+  end
```

Comment:
changes item active status to true

Reply:
This also might be part of the conflicts because I think Jordan had this method in his last PR.

Reply:
merchant.rb conflict (you can look through the conflict files if you want)
```
  def activate_items
    items.update(active?: true)
  end
<<<<<<< jg_us52-54_admin_merchant_user_pages

=======
>>>>>>> master
end
```
end of file. so... i'm not sure why this is a conflict

Reply:
Fixed this one too!